gcloud-java-logging/baseline/src/main/java/com/google/cloud/logging/spi/v2/ConfigServiceV2Api.java

```diff
@@ -48,33 +49,88 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
 
 // Manually-added imports: add custom (non-generated) imports after this point.
 
 // AUTO-GENERATED DOCUMENTATION AND SERVICE - see instructions at the top of the file for editing.
 /**
  * Service Description:
  *
+ * <p>This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * <pre>
+ * <code>
+ * try (ConfigServiceV2Api configServiceV2Api = ConfigServiceV2Api.createWithDefaults()) {
+ *   String projectName = "";
+ *   ListSinksResponse response = configServiceV2Api.listSinks(projectName);
+ * }
+ * </code>
+ * </pre>
+ *
+ * <p>Note: close() needs to be called on the configServiceV2Api object to clean up resources such
+ * as threads. In the example above, try-with-resources is used, which automatically calls
+ * close().
+ *
+ * <p>The surface of this class includes several types of Java methods for each of the API's methods:
+ *
+ * <ol>
+ * <li> A "flattened" method. With this type of method, the fields of the request type have been
+ * converted into function parameters. It may be the case that not all fields are available
+ * as parameters, and not every API method will have a flattened method entry point.
+ * <li> A "request object" method. This type of method only takes one parameter, a request
+ * object, which must be constructed before the call. Not every API method will have a request
+ * object method.
+ * <li> A "callable" method. This type of method takes no parameters and returns an immutable
+ * ApiCallable object, which can be used to initiate calls to the service.
+ * </ol>
+ *
+ * <p>See the individual methods for example code.
+ *
+ * <p>Many parameters require resource names to be formatted in a particular way. To assist
+ * with these names, this class includes a format method for each type of name, and additionally
+ * a parse method to extract the individual identifiers contained within names that are
+ * returned.
+ *
+ * <p>This class can be customized by passing in a custom instance of ConfigServiceV2Settings to
+ * create(). For example:
+ *
+ * <pre>
+ * <code>
+ * ConfigServiceV2Settings configServiceV2Settings = ConfigServiceV2Settings.defaultBuilder()
+ *     .provideChannelWith(myCredentials)
+ *     .build();
+ * ConfigServiceV2Api configServiceV2Api = ConfigServiceV2Api.create(configServiceV2Settings);
+ * </code>
+ * </pre>
+ *
```

Comment:
Why is the naming of classes in `logging` different from the naming of classes in `pubsub`?

In `pubsub` we have 
- `PublisherApi`, `SubscriberApi`
- `PublisherSettings`, `SubscriberSettings`

In `logging`:
- `ConfigServiceV2Api`, `LoggingServiceV2Api`, `MetricsServiceV2Api`
- `ConfigServiceV2Settings`, `LoggingServiceV2Settings`, `MetricsServiceV2Settings`

Is the `V2` in the name really necessary?